### ISSUE TITLE
Data Model

### ISSUE THREAD

**Issue by vhutheesing** [BIDS-collaborative/the-carpool-project#19]
@zhuwr0423 @luyangyi Could you both discuss at our meeting Friday how we can adapt the spreadsheet on google drive (the one that has my Moves data for September) to our data model?  Please let me know, thanks.  Some of the issues you should cover are:
1. how can the anonymous data be parsed in order to further protect the user ID's privacy
2. how can the data be cleansed/restructured before applying proximity filter to entire data set of trips
3. how can we show the data once we've applied a proximity filter to it?

**Comment by luyangyi** [comment]
We have discussed about the data model. I was wondering that whether we just consider about the situation when people have the same start location or not. If so, we do not need to take situations when people drive passing by someone and take him or her on into the consideration?

**Comment by vhutheesing** [comment]
@luyangyi @zhuwr0423 yes, that's correct Yangyi.  Driver's will only pick up people once, at the start of their trip, after taking one or two or three turns from where they started their trip.  Then, the only time they will have to stop during the rest of their trip, is to drop their riders off (so no picking up new riders).   If you like, I will explain more about this at our next meeting.

**Comment by luyangyi** [comment]
I see, thank you so much for the explanation.

**Comment by vhutheesing** [comment]
@luyangyi @zhuwr0423 I will be at BIDS from 3-5pm today in case you can join me for part of it, but i must leave at 5pm sharp.  Great if we can discuss the data model.  Please read Project Description on this page since I just updated it:   http://hackingmeasurement.berkeley.edu/projects/the-carpool-project.html
